sample_configs/values-gs.yaml

```diff
@@ -50,7 +54,7 @@ s3_annotations: &s3-ann
 
 compactor: *s3-ann
 
-ingester: 
+ingester:
   <<: *s3-ann
```

Comment:
Maybe it's fine to just copy-paste the annotation? Means duplication, but these yaml magic might also be a bit irritating for the end user. But just realizing this came in earlier. Also no strong opinion on this.

Reply:
Yeah I see your concern. I'll write something as a comment to clarify this.